guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage.adoc

```diff
@@ -0,0 +1,31 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage"]
+= Preparing for disaster recovery with active and passive {ProjectServer} with external storage
+
+Create your passive {ProjectServer} as a clone of your active {ProjectServer}.
+Make sure that both servers connect to the database content on your network-attached storage (NAS).
+
+.Procedure
+. Replicate the `/var/lib/pulp` and `{postgresql-lib-dir}` directories from the active {ProjectServer} to your NAS.
+// Do we want to keep this generic like the virtualization scenario, or is this a good place to provide example rsync commands or something?
+. Clone your active {ProjectServer}.
+See xref:cloning_satellite_server[].
++
+Keep the source server powered on and power off the new cloned server.
+The source server remains your primary active server, while the clone server becomes the secondary passive server.
+. Determine how you want to attach the database content on the NAS to your passive server:
+* If you mount the storage directly by both your active and passive server, the servers will always see the same, up-to-date content.
+* If you mount the storage only to your active server and replicate the storage periodically to another location, the passive server will access the data only if it takes over as the active server.
+// This would be a good place to talk more about the pros and cons of these two alternatives, wouldn't it? What would be those pros and cons?
+
+.Verification
+Perform this test in an isolated staging environment:
+// Does this make sense? We don't want users testing this in production, right?
+
+. Stop the active server to mimic failure.
+// How? foreman-maintain service stop?
+. Start the passive server.
+// How? foreman-maintain service start?
```

Comment:
I can see two scenarios:

 1. A passive server that is running all the time but inaccessible. In this case the user would simply ensure storage is attached properly and flip DNS.
 2. A passive server that is something like an image, that gets turned up, storage attached and then update is run.
 3. A machine that is spun up, restore is run with storage attached.

@evgeni Vet me here

Reply:
1. you can't count ;-)
2. when you run active/passive your nodes need to be identical and the only change you perform is "start the service" (however that is exactly defined)
3. I guess it should be something like your 1, but "inaccessible" means "services are down" (even if the machine is not accessed, running services would mean workers pick up work from the pool etc, not what you want!)

Reply:
I'm wondering whether all of this goes into too much detail for a testing/verification procedure. As an alternative, I could also drop line 27, which would take users directly to line 29 to tell them 'verify your passive server', however they would decide to do it. Is that just me being lazy, or would it actually make sense like this?

Reply:
Experience tells me that these details are questions users will ask, and this guide is intended to help reduce the questions being asked by codifying our knowledge. Perhaps we say as @evgeni did in #3 - by stating inaccessible and then giving some examples.